projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1322,6 +1323,24 @@ private boolean getAnswer(
     return parameters;
   }
 
+  /**
+   * Get information of the container(first element in {@code parameters}).
+   * <p> </p>
+   * Returns true if successfully get container information, false otherwise
```

Comment:
```
Get information of the container (first element in {@code parameters}).

<p>Returns {@code true} if successfully get container information, {@code false} otherwise.
```